modules/npcs/src/main/java/eu/cloudnetservice/modules/npc/platform/bukkit/command/NPCCommand.java

```diff
@@ -580,6 +582,72 @@ public boolean onCommand(CommandSender sender, Command command, String label, St
         // sets the target group of the npc
         case "tg", "targetgroup" -> updatedNpc = NPC.builder(npc).targetGroup(args[2]).build();
 
+        case "r", "rot", "rotate" -> {
+          if (args.length < 4) {
+            sender.sendMessage("§cInvalid usage! Use §6/cn edit rotate <yaw/pitch> <value>§c!");
+            return true;
+          }
+
+          // we want to use ~ as an operator to indicate whether the provided value is relative or absolute
+          var input = args[3];
+          var relative = input.startsWith("~");
+          if (relative) {
+            input = input.substring(1);
+          }
+
+          var position = npc.location();
+          var value = Doubles.tryParse(input);
+          if (value == null) {
+            sender.sendMessage(String.format("§cUnable to parse value from string §6%s§c.", args[3]));
```

Comment:
Maybe we should use something like `Unable to parse yaw or pitch from input %s`?